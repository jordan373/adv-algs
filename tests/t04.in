6
addEdge 0 1 2
addEdge 0 2 3
addEdge 1 2 1
addEdge 1 3 7
addEdge 1 4 1
addEdge 2 4 3
addEdge 3 4 5
addEdge 3 5 2
addEdge 4 5 2
printGraph
dijkstra 3
printDistance
printParents
printPath 3 0
lenghtShortestW 3 0
