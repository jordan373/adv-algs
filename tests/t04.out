Graph is:
1 2 
0 2 3 4 
0 1 4 
1 4 5 
1 2 3 5 
3 4 
Running the Dijkstra's algorithm on 3
Distances:
7 5 6 0 4 2 
Parents:
1 4 1 3 5 3 
Path from 3 to 0 is:
3 5 4 1 0 
Calculating length of the shortest path from 3 to 0
3 5 4 1 0 
The length is 7
